imgix-rails.gemspec

```
@@ -36,7 +36,7 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency "imgix", "~> 3.0"
 
   spec.add_development_dependency "bundler", ">=1.9"
-  spec.add_development_dependency "rake", "~> 10.0"
+  spec.add_development_dependency "rake", ">= 12.3.3"
```

Should we make this a little more restricted?
```suggestion
  spec.add_development_dependency "rake", "~> 12.3"
```